packages/protobuf-test/package.json

```diff
@@ -13,6 +13,7 @@
     "generate:js": "protoc --es_out=src/gen/js --es_opt=ts_nocheck=false,target=js+dts,import_extension=.js --proto_path=. $(buf ls-files extra) --proto_path=$(upstream-include test) $(upstream-files test) google/protobuf/type.proto",
     "postgenerate": "license-header src/gen",
     "perf": "tsx src/perf.ts benchmark '.*'",
+    "profile": "dexnode dist/esm/perf.js run 'fromBinary perf-payload.bin' 10000",
```

Comment:
The resulting logs can be inspected with the [deopt explorer](https://github.com/microsoft/deoptexplorer-vscode). Seems worth keeping this script and dep.